source/docs/articles/sites/code/developing-directly-with-sftp-mode.md

```diff
@@ -17,7 +17,7 @@ Due to the nature of our platform, the connection information will change from t
 
  ![Enable SFTP mode](/source/docs/assets/images/desk_images/278855.png)
 <div class="alert alert-danger" role="alert"><h4>Warning</h4>
-SFTP changes to code that have not been committed <em>will not be saved in backups</em> as they are not part of your code repository yet. If you have pending changes and the site is migrated (which we may do at any time for various reasons), your changes will be lost without warning. You must commit these file changes if you want to keep them permanently. If you switch from SFTP to Git mode while uncommitted changes are pending, you will be presented with a dialog confirming that you don't want to keep these changes, and if confirmed, they will be deleted.</div>

```

Comment:
I'd like to get more feedback on whether this is no longer true. @newtoid I saw you commented in Slack earlier. Do you remember where you heard that from?